bridge/functions.php

```diff
@@ -35,13 +35,19 @@ function dd_tracing_enabled()
 /**
  * Checks if any of the provided classes exists.
  *
- * @param string[] $sentinelClasses
+ * @param string[]|string $sentinelClasses
  * @return bool
  */
-function any_class_exists(array $sentinelClasses)
+function any_class_exists($sentinelClasses)
```

Comment:
I can't find where `any_class_exists()` is being invoked from. Where is that? :)

Also - instead of changing the method signature, could we just invoke it differently with `any_class_exists(['FooClass'])`?

Reply:
Indeed these seem to be from older code, probably it usage was removed recently. Will double check the tests results maybe its possible I "fixed" wrong function 😮

Reply:
Oh I misread the error - assumed that it was related to the other autoloader errror - however the failing tests was simply executing every defined function 😮